task/s2i-angular-nginx/0.1/s2i-angular-nginx.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  annotations:
+    tekton.dev/displayName: s2i angular nginx
+    tekton.dev/categories: Image Build
+    tekton.dev/tags: image, build, angular, nginx
+    tekton.dev/pipelines.minVersion: '0.19'
+  name: s2i-angular-nginx
+  labels:
+    app.kubernetes.io/version: '0.1'
+spec:
+  params:
+    - default: '14'
+      description: The tag of nodejs version
+      name: NODEJS_VERSION
+      type: string
+    - default: .
+      description: The location of the path to run s2i from.
+      name: CONTEXT_PATH
+      type: string
+    - default: 'true'
+      description: >-
+        Verify the TLS on the registry endpoint (for push/pull to a non-TLS
+        registry)
+      name: TLSVERIFY
+      type: string
+    - description: Location of the repo where image has to be pushed
+      name: IMAGE
+      type: string
+    - default: >-
+        registry.redhat.io/rhel8/buildah@sha256:e19cf23d5f1e0608f5a897f0a50448beb9f8387031cca49c7487ec71bd91c4d3
+      description: The location of the buildah builder image.
+      name: BUILDER_IMAGE
+      type: string
+  results:
+    - description: Digest of the image just built.
+      name: IMAGE_DIGEST
+  steps:
+    - image: registry.access.redhat.com/ubi8:latest
```

Comment:
instead of using `latest` tag, can you please pin the image with appropriate tag+digest? eg `registry.access.redhat.com/ubi8:8.3@sha256:someshadigest`